erdpy_network/contract_query_response.py

```diff
@@ -13,7 +13,7 @@ def from_http_response(payload: Dict[str, Any]) -> 'ContractQueryResponse':
         return_data = payload.get('returnData', []) or payload.get('ReturnData', [])
         return_code = payload.get('returnCode', '') or payload.get('ReturnCode', '')
         return_message = payload.get('returnMessage', '') or payload.get('ReturnMessage', '')
-        gas_remaining = payload.get('gasRemaining', 0) or payload.get('GasRemaining', 0)
+        # gas_remaining = payload.get('gasRemaining', 0) or payload.get('GasRemaining', 0)
```

Comment:
remove commented code?

Reply:
It's used now to compute the gas used. Same as erdjs-network-providers.